### PR TITLE
This is for collecting data

### PR DESCRIPTION
**Pull request by Gorott**
just for collecting data nothing else

DO NOT DELETE BRANCH!!!!!!!!!!!!!!!!!!

### CODE REVIEW

File: events/slashCmds.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const l10n = require("../l10n")
 
 module.exports = (client) => {
     client.on("interactionCreate", async (interaction) => {
+        console.log("ello")
         if (!interaction.isCommand()) return
+        console.log("ello")
```
Comment (Hackante) [inline events/slashCmds.js:12]: Do you still need them for debugging?

File: commands/economy/profile.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { players } = require("../../db.js")
+/*const { players } = require("../../db.js")
```
Comment (Hackante) [inline commands/economy/profile.js:1]: remove the comment or is the profile command not working anymore? 🤔

File: server.js [hunk 3]
```diff
@@ -305,6 +306,6 @@ if (typeof maint == "string" && maint.startsWith("config-")) {
 
 client.on("error", (e) => console.error(e))
 
-client.login(process.env.TOKEN)
+client.login(process.env.TOKENs)
```
Comment (Hackante) [inline server.js:309]: Is that for beta .env ?

File: server.js [hunk 4]
```diff
@@ -207,7 +208,7 @@ client.on("ready", async () => {
     client.user.setActivity(client.user.username.toLowerCase().includes("beta") ? "testes gae on branch " + branch + " and commit " + commit : "Wolvesville Simulation!")
     console.log("Connected!")
     client.userEmojis = client.emojis.cache.filter((x) => config.ids.emojis.includes(x.guild.id))
-    client.channels.cache.get("832884582315458570").send(`Bot has started, running commit \`${commit}\` on branch \`${branch}\``)
+    // client.channels.cache.get("832884582315458570").send(`Bot has started, running commit \`${commit}\` on branch \`${branch}\``)
```
Comment (Hackante) [inline server.js:211]: You can add that again if that doesn't cause any errors

File: server.js [hunk 5]
```diff
@@ -71,6 +71,7 @@ fs.readdir("./slashCommands/", (err, files) => {
                 props.category = file
                 try {
                     client.slashCommands.set(props.command.name, props)
+                    console.log(client.slashCommands)
```
Comment (Hackante) [inline server.js:74]: For debugging?

File: package.json [hunk 6]
```diff
@@ -23,7 +23,8 @@
         "array-pull": "^0.2.1",
         "axios": "^0.27.2",
         "callsite-record": "^4.1.3",
-        "canvas": "^2.9.0",
```
Comment (Hackante) [inline package.json:26]: When you add the profile command also add canvas again

**Review by Hackante — CHANGES_REQUESTED** [review]

**Comment by Gorott** [comment]
made the changes

**Review by Hackante — APPROVED** [review]

**Pull request merged**